app/assets/stylesheets/application.scss

```diff
@@ -4,6 +4,10 @@ $baseFontSize: 12px;
 @import "bootstrap";
 @import "bootstrap-responsive";
 @import "font-awesome";
+// Temporary fix until the chosen gem is fixed
+// https://github.com/tsechingho/chosen-rails/issues/97
+$chosen-sprite: image-url("chosen-sprite.png");
```

Comment:
Huh, is this really the best place for this, or are we okay with putting this here because it's intended as strictly temporary?

Reply:
Yeah, I put it here because it's meant to be temporary, and a fairly loud place to say: hey, check if this is fixed yet; if so get rid of me. If it were meant to be permanent I would find a better home for it.